src/plugin.cc

```diff
@@ -725,3 +700,46 @@ void plugin::Zeek_Spicy::Plugin::autoDiscoverModules() {
         searchModules(zeek::util::zeek_plugin_path());
     }
 }
+
+void plugin::Zeek_Spicy::Plugin::disableReplacedAnalyzers() {
+    for ( auto& info : _protocol_analyzers_by_type ) {
+        if ( info.name_replaces.empty() )
+            continue;
+
+        auto tag = ::zeek::analyzer_mgr->GetAnalyzerTag(info.name_replaces.c_str());
+        if ( ! tag ) {
+            ZEEK_DEBUG(hilti::rt::fmt("%s is supposed to replace protocol analyzer %s, but that does not exist",
+                                      info.name_analyzer, info.name_replaces));
+
+            continue;
+        }
+
+        ZEEK_DEBUG(hilti::rt::fmt("  Replaces existing protocol analyzer %s", info.name_replaces));
+        info.replaces = tag;
+        ::zeek::analyzer_mgr->DisableAnalyzer(tag);
```

Comment:
Just as a note, before this change we would potentially replace less analyzers than expected while now we should always disable the replaced analyzer. That makes it possible to e.g., have two Spicy analyzers which replace each other so that with this impl none would be active. We could add a validation step to reject that.

Same below.

Reply:
Seems fine to me to leave as is.